chat.go

```diff
@@ -26,23 +27,50 @@ type ChatCompletionMessage struct {
 	// the documentation for the official library for python:
```

Comment:
`name` now is in the official documentation: https://platform.openai.com/docs/api-reference/chat/create#chat/create-name